local_setup/quickstart_server.py

```diff
@@ -157,9 +157,14 @@ async def get_all_agents():
         
         if not agent_keys:
             return {"agents": []}  
-
-
-        agents_data = await asyncio.gather(*(redis_client.get(key) for key in agent_keys))
+        agents_data = []
+        for key in agent_keys:
+            try:
+                data = await redis_client.get(key)
+                agents_data.append(data)
+            except Exception as e:
+                logger.error(f"An error occurred with key {key}: {e}")
+                agents_data.append(None)
```

Comment:
@khushaljethava Can youo remove adding `None` to the array in case of exception. I don't think it'll server any purpose.

Reply:
@prateeksachan please check.